emission/core/wrapper/suggestion_sys.py

```diff
@@ -6,7 +6,9 @@
 import json
 import logging
 import re
-import emission.storage.timeseries.abstract_timeseries as esta
+import sys
+
+#import emission.storage.timeseries.abstract_timeseries as esta
```

Comment:
> NOTE #2: If you comment out the line
"import emission.storage.timeseries.abstract_timeseries as esta"
in suggestion_sys, you don't need to have the database running. Please make
sure not to commit this change